flags.go

```diff
@@ -50,10 +50,15 @@ var (
 		Name:  "tlsverify",
 		Usage: "Use TLS and verify the remote",
 	}
+	flOverCommit = cli.IntFlag{
+		Name:  "overcommit, oc",
+		Usage: "OverCommit to apply on resouces",
```

Comment:
s/resouces/resources/g
Also is this the help text that's going to show up in the cli? If it is I think more description what and why are we overcommiting will be nice.